android/README.md

```diff
@@ -126,48 +126,14 @@ Please see [these instructions](docs/guides/installation.md) for installing and
 
 ## Adding a new Backend
 
-### Backend changes
-In mobile_back_vendor, the following steps are required to add your backend:
-
-1. Create a branch in mobile_back_[vendor] that builds your backend using cpp/c/backend_c.h
-2. Generate an artifacts dir with:
-  a. BUILD
-  b. libbackendname.so shared library
-  c. dependent shared libraries (e.g. libaccelerator.so, libMySDK.so)
-
-Use the following as a template for the layout of the artifacts dir
-replacing _vendor_ with the actual vendor/backend name:
-```
-artifacts
-├── BUILD
-└─── lib_arm64/
-    ├── libaccelerator.so
-    ├── libvendorbackend.so
-    └── libMySDK.so
-```
-The generated BUILD file is:
-```
-package(
-    default_visibility = ["//visibility:public"],
-    licenses = ["notice"],  # Apache 2.0
-)
+The following steps are required to add your backend:
 
-config_setting(
-    name = "android_arm64",
-    values = {"cpu": "arm64-v8a"},
-)
+1. Create a mobile_back_[vendor] top level directory that builds your backend using android/cpp/c/backend_c.h
+2. Implement all the C API backend interfaces defined in android/cpp/c/backend_c.h
+3. Add a bazel BUILD file to create lib[vendor]backend.so shared library
```

Comment:
Is this a mandatory requirement that the build has to be with Bazel . Can we use Cmake instead ?